### DISCUSSION TITLE
CIDER complains of the cider-nrepl version

### DISCUSSION THREAD

**Discussion by jgomo3** [clojure-emacs/cider#3324] [Q&A] [answered]
Yeap, I upgraded CIDER and it happened to me: "CIDER complains of the cider-nrepl version"

Specifically, I [see a number like X.X.X, and I'm starting the REPL with cider-jack-in](https://docs.cider.mx/cider/1.6/troubleshooting.html#you-see-a-number-like-x-x-x-and-youre-starting-the-repl-with-cider-jack-in), and contrary to what the F.A.Q. says, I'm not "manually adding the cider-nrepl middleware in your project": This is a dummy project created with `clj-new`, with no mention of `cider` in the `deps.edn` file.

The message I'm having is:

"WARNING: CIDER 1.6.0 requires cider-nrepl 0.30.0, but you're currently using cider-nrepl 0.28.3. The version mismatch might break some functionality!"

So I suppose that I can solve this by simply upgrading `cider-nrepl` from `0.28.3` to `0.30.0`: How can I do that? I can't find `cider-nrepl` in the Emacs Package Manager.

**Comment by bbatsov** [comment, marked as answer]
`cider-nrepl` is not an Emacs package. The version that's injected is based on the version of CIDER you're using. I assume that you didn't restart Emacs after updating your packages, right? Otherwise I can't imagine how you'd be getting such a warning.

**Reply by jgomo3** [reply]
I'm not been able to replicate. Thanks!
